setup.py

```diff
@@ -52,7 +52,7 @@
 DOWNLOAD_URL = "https://github.com/pydicom/pydicom/archive/master.zip"
 LICENSE = "MIT"
 VERSION = __version__
-# PACKAGE_DATA = {'pydicom': ['../tests/tests_files/*']}
+PACKAGE_DATA = {'pydicom': ['tests/test_files/*', 'tests/charset_files/*']}
 REQUIRES = []
```

Comment:
I would give an example. In scikit-learn, we have a bug which only appear 1 over 5 times in Mac OS X. The CI could not detecting those since we don't usually test 5 times in a row the package.
A pypi user could still be able to report you the testing error even without having the source.
In some way it could ease the maintenance and reporting. Does it convince you?